### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Connect to Local Server_Zooming: When browser is resized to 200%, some of the contents and controls disappears and they no longer remains accessible for the user.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#101]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS1.4.4

**Repro Steps:**
1. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2. Download and install "Microsoft IIS Administration" application.
3. Tab till "Skip This" button and Press Enter.
4. set your browser zooming till 200% and verify that all the contents and controls on the page are visible and accessible.

**Actual Result:**
When browser is resized to 200%, following contents and controls disappears and they no longer remains accessible for the user:
- "Remote Server or manual setup" and "Get Microsoft IIS Administration" buttons.
- "Connect" heading.

**Expected Result:**
All the contents and controls of the page should be visible and accessible even after zooming the screen to 200%.

**User Impact:**
Visually Impaired users who uses resize features to enlarge the text on the page, will not be able to access all the controls of the page.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Attachment:**
![bug 101_zooming issue](https://user-images.githubusercontent.com/38521133/44656831-2581ca00-aa18-11e8-89bc-7e66a0009a38.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Arjun553** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #C+E-H; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS2.1.1; #Resize; #.NETWebsites; #Edge; #Chrome

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
